package.json

```diff
@@ -1,36 +1,55 @@
 {
-  "name": "bench-template",
+  "name": "fastify-slow-down",
   "version": "1.0.0",
-  "description": "Template to start a new repository on the bench",
+  "description": "A slow down plugin for fastify",
   "main": "index.js",
   "type": "module",
+  "files": [
+    "index.js",
+    "lib"
+  ],
   "scripts": {
     "lint": "eslint .",
-    "test": "echo \"Error: no test specified\" && exit 0",
+    "test": "tap test/*.test.js",
```

Comment:
tap doesn't do coverage for es modules. use c8. see how it's done here https://github.com/nearform/fastify-ravendb/blob/master/package.json#L31. also make sure we have 100% coverage please